config/locales/client.en.yml

```diff
@@ -25,3 +25,14 @@ en:
           help_text:
             email_available: "Reach bottom right of the page to see this Patreon's pledge history."
             email_not_available: "Select the Patreon member and reach bottom right of the page to see the pledge history."
+      donation_prompt:
+        body: |
+          <h1>Donation Prompt Title</h1>
+
+          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
```

Comment:
I don't think we should be using lorem ipsum but instead provide either an example of donation prompt or explain to the admin what to put here?

Reply:
https://github.com/muhlisbc/discourse-patreon/commit/ac157bbc7f8620f6c66cf031a3c36f60936202dc

Suggestions are welcomed.